commands-yml/commands/session/other/get-log-types.yml

```diff
@@ -0,0 +1,73 @@
+---
+name: Get available log types
+short_description: Get the log for a given log type. Log buffer is reset after each request
+   
+example_usage:
+  java:
+    |
+      Set<String> logEntries = driver.getLogTypes();
```

Comment:
@dpgraham Selenium has it implemented but i don't think so it may work here.

Reply:
It should work. Appium implements it on the server side.

Reply:
I don't see we have it on java-client though. Candidate to implement at client side? @imurchie

Reply:
Sure. I thought it was part of the Selenium client. But if it isn't, it ought to be implemented (whatever the name is).

Reply:
Ok i will check it out.

Reply:
It should be `driver.manage().logs().getAvailableLogTypes();`